demo-v1/src/components/App.tsx

```diff
@@ -66,8 +66,9 @@ function App() {
 
   const handleEndCall = () => {
     stopTimer();
-    cti.callAnswered();
-    cti.callEnded();
+    cti.callCompleted({
```

Comment:
Just a heads up that this closes the widget and we can't get to the next screen to "save call" unless you reopen the widget.

Reply:
Yeah I think we should invoke `callCompleted` on "save call" instead. Added the change in https://github.com/HubSpot/calling-extensions-sdk/pull/78/commits/c6b44362f66fb16af5d5701ec3d9d5208d2d5a70